inference-engine/thirdparty/clDNN/tests/test_cases/fusings_gpu_test.cpp

```diff
@@ -7791,3 +7793,247 @@ INSTANTIATE_TEST_CASE_P(fusings_gpu, scatter_nd_update_scale_activation_eltwise,
+        network network_not_fused(this->engine, this->topology_non_fused, bo_not_fused);
+        network network_fused(this->engine, this->topology_fused, bo_fused);
+        network_fused.set_input_data("input", input_prim);
+        network_not_fused.set_input_data("input", input_prim);
+        compare(network_not_fused, network_fused, p);
+    }
+
+    layout get_input_layout(gather_nd_test_params& p) {
+        return layout{ p.data_type, p.input_format, p.input_shape };
+    }
+
+    layout get_indices_layout(gather_nd_test_params& p) {
+        return layout{ p.data_type, p.indices_format, p.indices_shape };
+    }
+
+    layout get_output_layout(gather_nd_test_params& p) {
+        return layout{ p.data_type, p.output_format, p.output_shape };
+    }
+
+    layout get_per_channel_layout(gather_nd_test_params& p) {
+        return layout{ p.default_type, p.default_format, tensor{1, p.output_shape.feature[0], 1, 1} };
+    }
+};
+
+class gather_nd_quantize : public GatherNDPrimitiveFusingTest {};
+TEST_P(gather_nd_quantize, basic) {
+    auto p = GetParam();
+    create_topologies(input_layout("input", get_input_layout(p)),
+        data("gather_nd_indices", get_mem(get_indices_layout(p), 0, p.max_number_in_indices - 1)),
+        data("in_lo", get_mem(get_per_channel_layout(p), min_random, 0)),
+        data("in_hi", get_mem(get_per_channel_layout(p), 1, max_random)),
+        data("out_lo", get_mem(get_single_element_layout(p), -127)),
+        data("out_hi", get_mem(get_single_element_layout(p), 127)),
+        gather_nd("gather_nd_prim", "input", "gather_nd_indices", p.indices_rank, p.batch_dims),
+        quantize("quantize", "gather_nd_prim", "in_lo", "in_hi", "out_lo", "out_hi", 255, data_types::i8),
+        reorder("reorder_bfyx", "quantize", p.default_format, data_types::f32)
+    );
+    tolerance = 1.f;
+    execute(p);
+}
+
+INSTANTIATE_TEST_CASE_P(fusings_gpu, gather_nd_quantize,
+    ::testing::ValuesIn(std::vector<gather_nd_test_params>{
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_4D_1, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_4D_2, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_4D_3, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_4D_4, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_4D_5, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_4D_6, 2, 3 },
+
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_5D_1, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_5D_2, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_5D_3, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_5D_4, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_5D_5, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_5D_6, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_5D_7, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_5D_8, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_5D_9, 2, 3 },
```

Comment:
I think count of test cases can be reduced without loss of coverage.

Reply:
I applied it.